src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,61 @@
+$elements-margin: 140px;
+
+.watch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  @include square;
```

Comment:
I'm not certain if it works without specifying the size. Have you tested it since making the changes? Please check if it's functioning properly and deploy the updated code to gh-pages.